### PR TITLE
Add frint-props and frint-props-react to the sidebar

### LINKED ISSUE #12: Include `frint-props` and `frint-props-react`
Include both the packages in docs sidebar.

### PR DESCRIPTION
**Pull request by maryam-saeidi**
Closes #12

### CODE REVIEW

File: src/pages/docs/packages/frint-props-react.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: frint-props-react
+importContentFromUrl: "https://raw.githubusercontent.com/frintjs/frint-props/master/packages/frint-props-react/README.md"
```
Comment (fahad19) [inline src/pages/docs/packages/frint-props-react.md]: I am thinking if we can have a generic key with paths like this, for all the markdown imports:

```yaml
importFromGitHub: "frintjs/frint-props/master/packages/frint-props-react/README.md"
```

File: src/static/data/sidebar.json [hunk 2]
```diff
@@ -8,156 +8,164 @@
...
       "label": "Changelog",
       "to": "/docs/changelog"
     },{
-      "id": "project_3",
+      "id": "project_contribution_guide",
       "label": "Contribution guide",
       "to": "/docs/contributing/guide"
     },{
-      "id": "project_4",
+      "id": "project_maintainers_guide",
```
Comment (fahad19) [inline src/static/data/sidebar.json]: very nice regarding moving away from numbered keys 👍

Comment (maryam-saeidi) [reply src/static/data/sidebar.json]: Thanks :)

File: src/pages/docs/packages/frint-props-react.md (hunk 1 above)
Comment (maryam-saeidi) [reply src/pages/docs/packages/frint-props-react.md]: Yeah, That sounds good, I'll apply it.

**Review by fahad19 — APPROVED** [review]

**Comment by fahad19** [comment]
Thanks @maryam-saeidi! 💖

**Pull request merged**